Todo_TS/types/todoTypes.ts

```diff
@@ -0,0 +1,36 @@
+export interface IApp {
+  $target: HTMLElement | null;
+  initialState: [];
+}
+export interface ITodoHeader {
+  $target: HTMLElement | null;
+  text: string;
+}
+export interface ITodoForm {
+  $target: HTMLElement | null;
+  onSubmit: (text: string) => void;
+}
+export interface ITodoCount {
+  $target: HTMLElement | null;
+  initialState: [];
+}
+
+export interface ITodoList {
+  $target: HTMLElement | null;
+  initialState: [];
```

Comment:
P5: State에 대한 요소 값이 Todo 값들이 들어오니 `ITodo[]`로 타입을 명시해줘도 될 거 같아요.
```suggestion
  initialState: ITodo[];
```